src/fillTank.test.js

```diff
@@ -1,11 +1,152 @@
-'use strict';
+/* eslint quotes: ["off"] */
+"use strict";
 
-describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+describe("fillTank", () => {
+  const { fillTank } = require("./fillTank");
 
-  it('should ', () => {
+  it(`should not fill tank, because it is
+     already full (not given amount)`, () => {
```

Comment:
The test description mentions 'not given amount', but the `fillTank` function is called with an amount of 50. Ensure the test description matches the test logic.